packages/@aws-cdk/aws-synthetics/test/integ.asset.expected.json

```diff
@@ -1,356 +0,0 @@
-{
```

Comment:
LOL! Thanks for deleting this

Reply:
Np haha. When I was breaking out the other, I noticed this had no matching test :)